### DISCUSSION TITLE
Fuente con ligaturas

### DISCUSSION THREAD

**Discussion by andres-requena-uneat** [mmasias/IdSw2-22-23#3] [Ideas]
## [FiraCode](https://github.com/tonsky/FiraCode/blob/master/LEEME.md). 
Ahí mismo tiene links mostrando como instalarlo junto con listas de compatibilidad con diferentes IDEs y Terminales.

**Comment by mmasias** [comment]
Instalada! Está muy chula: ¡gracias!
